addon/webextension/selector/ui.js

```diff
@@ -231,7 +231,11 @@ window.ui = (function () { // eslint-disable-line no-unused-vars
                <body>
                  <div class="preview-overlay">
                    <div class="fixed-container">
-                     <div class="preview-instructions"></div>
+                     <div class="preview-instructions">
+                       <span class="preview-instructions-step-one"></span>
+                       &nbsp;
+                       <span class="preview-instructions-step-two"></span>
```

Comment:
The English result with forced RTL direction is expected, period is a weak character and follows RTL, the rest is LTR.

The real question is, why do we need to split this string in two? Can't we just use one? That would also solve the awkward wrapping.